cg/meta/demultiplex/utils.py

```diff
@@ -206,11 +207,18 @@ def parse_manifest_file(manifest_file: Path) -> list[Path]:
 
 def is_file_relevant_for_demultiplexing(file: Path) -> bool:
```

Comment:
Reversed the logic for this function, make sure that any missing files are indeed irrelevant